platforms/riscv32/devices.bzl

```diff
@@ -7,14 +7,14 @@ load("//config:device.bzl", "device_config")
 DEVICES = [
     device_config(
         name = "opentitan",
-        architecture = "rv32imc",
-        feature_set = "//platforms/riscv32/features:rv32imc-hardened",
+        architecture = "rv32imc_zba_zbb_zbc_zbs",
```

Comment:
Should this be the full expression or just `rv32imcb`?

Reply:
There is no umbrella B extension anymore :(